app/config/config.yml

```diff
@@ -67,12 +67,17 @@ doctrine:
         types:
             json: Sonata\Doctrine\Types\JsonType
             uuid_binary: Ramsey\Uuid\Doctrine\UuidBinaryType
+        mapping_types:
```

Comment:
Sorry, already removed. There was an old bug "Unknown database type enum requested, Doctrine\DBAL\Platforms\MySqlPlatform may not support it ".  It helped.